code/Other/DebugBot.cs

```diff
@@ -0,0 +1,111 @@
+#if DEBUG
+using Sandbox;
+using Sandbox.Internal;
+using System.Linq;
+
+namespace TTT;
+
+public partial class DebugBot : Bot
+{
+	/// <summary>
+	/// The Player pawn this bot controls.
+	/// </summary>
+	public Player Pawn;
+
+	/// <summary>
+	/// The host Player who spawned this bot.
+	/// </summary>
+	public Player Target;
+
+	public static bool Mimic;
+	public static bool Wander;
+
+	[ConVar.Replicated( "bot_debug" )]
+	public static bool DrawDebug { get; set; }
+
+	[ConCmd.Admin( "bot_mimic", Help = "Make bots mimic your inputs." )]
+	public static void ToggleMimicHost()
+	{
+		Mimic = !Mimic;
+		Wander = false;
+	}
+
+	[ConCmd.Admin( "bot_wander", Help = "Make bots randomly press move buttons." )]
+	public static void ToggleWander()
+	{
+		Wander = !Wander;
+		Mimic = false;
+	}
+
+	[ConCmd.Admin( "bot_reset", Help = "Resets bot to default settings." )]
+	public static void BotReset()
+	{
+		Wander = false;
+		Mimic = false;
+	}
+
+	[ConCmd.Admin( "bot_kill", Help = "Kills a bot by name if provided, else kills all bots." )]
+	public static void BotKill( string name = "" )
+	{
+		if ( string.IsNullOrEmpty( name ) )
+		{
+			foreach ( var b in All.ToArray() )
```

Comment:
Is this `Client.All`?

Reply:
It's Bot.All but it can be simplified to All

Reply:
//
		// Summary:
		//     Contains all bots on the server.
		public static IReadOnlyList<Bot> All => _all.AsReadOnly();